examples/templates/recorded_animation_svg_line_pie_charts/template.js

```diff
@@ -0,0 +1,480 @@
+            .text(d => d.label);
+
+        dotsSel.selectAll('.line')
+            .data(conf.lines)
+            .join(enter => enter.append('g').classed('line', true))
+            .attr('color', d => d.color)
+            .call(sel => sel.selectAll('circle')
+                .data(d => getCoordsFromLineConf(d))
+                .join('circle')
+                .attr('fill', 'currentColor')
+                .attr('r', 2)
+                .attr('cx', d => x(d.x))
+                .attr('cy', d => y(d.y))
+            )
+    }
+
+    render() {
+        return (
+            <div
+                className={this.props.className}
+                ref={node => this.containerNode = node}>
+
+                {this.state.width &&
+                    <SVG
+                        width={this.state.width + "px"}
+                        height={this.props.height + "px"}
+                        source={this.svgImg}
+                        init={(node) => {
+                            this.svgSel = select(node);
+                            const dotsSel = this.svgSel.select('#dots');
+
+                            this.svgSel.call(
+                                zoom().extent([
+                                    [0, 0],
+                                    [this.state.width, this.props.height]
+                                ])
+                                .translateExtent([
+                                    [
+                                        -0.5*this.state.width,
+                                        -0.5*this.props.height
+                                    ],
+                                    [
+                                        1.5*this.state.width,
+                                        1.5*this.props.height
+                                    ]
+                                ])
+                                .scaleExtent([0.9, 10])
+                                .on('zoom', () => {
+                                    this.svgSel.select('#content')
+                                        .attr("transform", event.transform);
+                                })
+                            );
+                        }}
+                        data={{
+                            dots: this.boundUpdateDots,
+                        }}
+                    />
+                }
+            </div>
+        );
+    }
+}
+
+const AnimatedLineChart = animated(LineChart);
+const AnimatedPieChart = animated(StaticPieChart);
+const AnimatedSVGChart = animated(SVGChart);
+
+
+class LineChartSection extends Component {
+    static propTypes = {
+        config: PropTypes.object.isRequired,
+    }
+
+    render() {
+        const config = {
+            yAxes: [{visible: true, belowMin: 0.1, aboveMax: 0.2}],
+            signalSets: this.props.config.sigSets
+        };
+
+
+        return (
+            <div className="container-fluid">
+                <AnimatedLineChart
```

Comment:
The LineChart breaks when its zoom is updated (try dragging the chart to the side or scrolling the mouse wheel). I suggest adding `withZoom={false}` to the properties. This possibly applies also to the other charts in the examples.